es6/report-generator.js

```diff
@@ -28,7 +28,8 @@ export function generateFileReport(file, spellingInfo) {
 
     const lineNumber = String(displayBlock.lineNumber);
     const lineNumberPadding = Array(10 - lineNumber.length).join(' ');
-    const linePrefix = `${lineNumberPadding}${lineNumber} |`;
+    const errorIndex = String(error.index);
+    const linePrefix = `${lineNumberPadding}${lineNumber}:${errorIndex} |`;
```

Comment:
thanks. The problem with this change is that we have padding so that lines are aligned - adding the error index invalidates that padding.
I would change it to

```
     const lineNumber = String(displayBlock.lineNumber);
     const errorIndex = String(error.index);
     const location = `${lineNumber}:${errorIndex}`;
     const locationPadding = Array(10 - location.length).join(' ');
     const linePrefix = `${locationPadding}${location} |`;
```